Makefile

```diff
@@ -77,3 +58,24 @@ format:
 .PHONY: tidy
 tidy:
 	go mod tidy
+
+.PHONY: check-goreleaser
+check-goreleaser:
+	@command -v goreleaser >/dev/null 2>&1 || { echo >&2 "goreleaser is required but not installed. Aborting. Run 'make install-goreleaser' to install"; exit 1; }
+
+.PHONY: install-goreleaser
+install-goreleaser:
+	@command -v goreleaser >/dev/null 2>&1 || { \
+		echo >&2 "goreleaser is required but not installed. Installing..."; \
+		curl -sfL https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh | sh; \
+	}
```

Comment:
Would it make sense to install goreleaser as part of tools.go?

Reply:
Not sure if we can do that. I'll look into it. But I think goreleaser doesn't work like that.  Totally 100% worth the exploration though.

Reply:
Also that url doesn't work :P

Reply:
HaH! We can! `go install github.com/goreleaser/goreleaser@latest` sweet!